docs/tags/index.md

```diff
@@ -0,0 +1,56 @@
+title: Tags 
+description: A Tag is an attribute of a data point (or metric) that could be used for grouping and filtering.
+
+Tags are sent by Sematext Agents as part of every data point and they are shown in UI as filters. 
+
+For example, as part of Docker container metrics, the agent sends hostname on which the container is running, container identifier, container name, container image as tags which appear as filters in UI. User can then group or filter the container metrics using these tags.
+
+At the moment all Common [Special Tags](../special-tags) apply to Monitoring Apps.
```

Comment:
We'll have to see what this feels like when published.  I see we have "Special Fields" under /docs/logs, and here we have Special Tags under /docs/tags.  A bit of asymmetry there...

Reply:
Hm yes agreed. @bsmid mentioned we should also have a tags section under Monitoring, which will link to the top-level tags page. I'll rename the Special Tags to Common Schema.